pkgs/development/python-modules/hap-python/default.nix

```diff
@@ -16,15 +16,15 @@
 
 buildPythonPackage rec {
   pname = "hap-python";
-  version = "4.6.0";
+  version = "4.7.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "ikalchev";
     repo = "HAP-python";
-    rev = "refs/tags/v${version}";
+    rev = "refs/tags/${version}";
     hash = "sha256-fAJB1gk8zTS/mW5KzWr3z26qctZc/EQlk//WM1Xwpl0=";
```

Comment:
This update is broken. Note how the hash didn't change. I suspect the upstream repo tagged incorrectly at first, then deleted and retagged after this update, because the upstream tag has the correct source now.